test/data/testSP.py

```diff
@@ -612,7 +608,7 @@ def testmetadata_symmetry_detected(self, data) -> None:
     @skipForParser("NBO", "attribute not implemented in this version")
     @skipForParser("Turbomole", "reading point group symmetry and name is not implemented")
     @skipForParser("xTB", "not implemented yet")
-    @skipForParser("Serenity", "not implemented yet")
+    @skipForParser("Serenity", "not applicable")
```

Comment:
If true, say instead something like
```suggestion
    @skipForParser("Serenity", "Serenity doesn't use symmetry")
```
because while this message isn't strictly necessary, it's a good habit to get into because sometimes skip reasons are more complicated.